csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,76 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
-    [DeploymentItem("chromedriver.exe")]
-    [DeploymentItem("geckodriver.exe")]
+   
     public class SampleTest
     {
+        // Instance to control the browser.
+        // The OpenQA.Selenium.IWebDriver interface is the main interface to use for testing, which represents an idealized web browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            // You should quit the driver on TearDown which will close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // First DataRow Attribute will create a new instance of the test method and pass Chrome as a paramter, so it'll test on ChromDriver
+        // Second DataRow Attribute will create a new instance of the test method and pass Firefox as a paramter, so it'll test on FirefoxDriver
+        // Instead of useing DataRow Attribute, you can create a helper fucntion that takes the the broswer type as a paremter
+        // Then call it twice with 2 differrent test methods for each browser type
         [DataRow(BrowserType.Chrome)]
         [DataRow(BrowserType.Firefox)]
+        // This test case shows a basic example: run a scan on an element, return the result, make sure it's as expected.
+        // This is the way to go if you have known/pre-existing violations you need to temporarily baseline.
         public void RunScanOnGivenElement(BrowserType browser)
         {
             this.InitDriver(browser);
             LoadTestPage();
 
-            AxeResult results = RunScanOnGivenElementBySelector("ul");
+            // Analyze the a known element in the page, and get the results
+            AxeResult results = RunScanOnGivenElementByTagName("ul");
+            // Assert that the number of violations that were found is that what you expect
             Assert.AreEqual(3, results.Violations.Length);
         }
 
-        private AxeResult RunScanOnGivenElementBySelector(string elementSelector)
+        private AxeResult RunScanOnGivenElementByTagName(string tagName)
         {
-            var selectedElement = _webDriver.FindElement(By.TagName(elementSelector));
+            // Find the first element matching the element tag name parameter
+            // * It acts as a sanity check that our browser automation setup basically works
```

Comment:
Are these bullet points misplaced? These would make more sense if it were happening as part of `LoadTestPage`, but here I think the `FindElement` is just necessary to get the argument to `_webDriver.Analyze`

Reply:
I just deleted it, it'd make more sense it we get the main element like we do in the typescript